src/js/controller/captions.js

```diff
@@ -303,6 +208,9 @@ define([
         this.setCaptionsList = function(captionsMenu) {
             _model.set('captionsList', captionsMenu);
         };
+
+        //this.loadTrack = _load;
+        this.isSDK = _isSDK;
```

Comment:
Are `this.isSDK` and `_isSDK` used anymore in captions.js? I see it again in the mixin, but looks like all other references were removed.